### PR TITLE
Add rubocop

### LINKED ISSUE #43: Linter
Hi,

In [this PR](https://github.com/infrablocks/ruby_terraform/pull/41/files) the changes should have been just the flag and not removing trailing spaces. I noticed it too late and it made me think if it would be ok to have some rules (e.g. `Rubocop`) in place.

### PR DESCRIPTION
**Pull request by jesusbv**
This Fixes #43

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Offense count: 1
+# Configuration parameters: CountComments, ExcludedMethods.
+# ExcludedMethods: refine
+Metrics/BlockLength:
+  Max: 27
+
+# Offense count: 8
+Metrics/CyclomaticComplexity:
+  Max: 14
```
Comment (jesusbv) [inline .rubocop_todo.yml:17]: Still some offenses to fix.
I think it's better to have a conversation and get feedback about those, though.

**Comment by tobyclemson** [comment]
Wow, thanks for this!

**Comment by tobyclemson** [comment]
Hey @jesusbv,

Sorry I still haven't got this merged in, will look to do so over the weekend.

Thanks,
Toby

**Comment by tobyclemson** [comment]
@jesusbv This has all been incorporated in master now so will close this PR. Thanks again!

**Pull request closed without merging**